book/website/afterword/glossary.md

```diff
@@ -570,6 +635,11 @@ Shared authorship
 Smoke Testing
   Very brief initial checks that ensure the basic requirements required to run the project hold. If these fail there is no point proceeding to additional levels of testing until they are fixed.
 
+Source Available
+  Where the source code of a piece of software is made available but not under the terms of an {term}`open license<Open License>`, you may have to pay to access the source code and agree to terms which prohibit its redistribution, or the code may be generally available and only prohibit commercial redistribution.
+  Source available code is proprietary but not closed source.
+  Whilst Generaly used in the context of software it can apply in other contexts such as {term}`open hardware<Open Source Hardware>`.
```

Comment:
```suggestion
  Whilst generally used in the context of software it can apply in other contexts such as {term}`open hardware<Open Source Hardware>`.
```